cypress/integration/roamingTest.js

```diff
@@ -1,16 +1,16 @@
-import RoumingPage from '../../pages/RoamingPage'
+import RoamingPage from '../../pages/RoamingPage'
 import * as data from '../../test_data/roaming_data'
```

Comment:
Regarding data import.
You need to name your import more detailed:
import * as homeData from '../../test_data/home_data'